docs/src/app/app-routes.jsx

```diff
@@ -103,6 +104,7 @@ const AppRoutes = (
       <Route path="refresh-indicator" component={RefreshIndicator} />
       <Route path="select-field" component={SelectField} />
       <Route path="sliders" component={Sliders} />
+      <Route path="svg-icon" component={SVGIconPage} />
```

Comment:
I'm not sure about the capital `SVGIcon`, the component displayName is `SvgIcon`.
Shouldn't we follow this?